src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -288,6 +289,8 @@ function MoneyRequestConfirmPage(props) {
                     props.iou.category,
                     props.iou.tag,
                     props.iou.merchant,
+                    props.policyTags,
+                    props.policyCategories,
```

Comment:
The orders are not correct.
Should be `currency, merchant, category, tag, billable`